TrackLog.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace OpenFindaBLE.Backend.Models
+{
+	public class TrackLog
```

Comment:
`BaseEntity` から派生する形にしてもらえると`BaseEntity` に存在するものをすべて含んだ扱いになるので`BaseEntity`を含んだ形にしてください

```suggestion
	public class TrackLog : BaseEntity
```